runtime/str.go

```diff
@@ -205,6 +205,37 @@ func strEq(f *Frame, v, w *Object) (*Object, *BaseException) {
 	return strCompare(v, w, False, True, False), nil
 }
 
+func strFind(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+	// TODO: Support unicode sub string
+	expectedTypes := []*Type{StrType, StrType, IntType, IntType}
+	argc := len(args)
+	if argc == 2 || argc == 3 {
+		expectedTypes = expectedTypes[:argc]
+	}
+	if raised := checkMethodArgs(f, "find/index", args, expectedTypes...); raised != nil {
+		return nil, raised
+	}
+	s := toStrUnsafe(args[0]).Value()
+	l := len(s)
+	start, end := 0, l
+	if argc >= 3 {
+		start = adjustIndex(toIntUnsafe(args[2]).Value(), l)
```

Comment:
Ok, will add a TODO. I need to learn a bit more about using slots.
Didn't know about IndexInt, thank you.